monero/wallet_client.go

```diff
@@ -496,7 +496,7 @@ func launchMoneroWalletRPCChild(walletRPCBin string, walletRPCBinArgs ...string)
 	cmd.Env = append(os.Environ(), "LANG=C", "LC_ALL=C", "TERM=dumb")
 
 	cmd.SysProcAttr = &syscall.SysProcAttr{
-		Pdeathsig: syscall.SIGTERM,
+		//Pdeathsig: syscall.SIGTERM,
```

Comment:
@dimalinux I had to comment this out since it doesn't build for `GOOS=darwin`

Reply:
The goal here is that if `swapd` (or a unit test for that matter) is killed in a way that doesn't shut down the wallet, the Linux OS will send SIGTERM to the `monero-wallet-rpc` process. You can read more about it under `PR_SET_PDEATHSIG` in the `prctl` man page. We could separate out the `SysProcAttr` handling into different files with different OS build tags.